ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -671,6 +672,20 @@ def _remove_lock_file(self):
                         os.remove(lock_file)
                     except OSError:
                         pass
+    
+    def _remove_mylocalip_file(self):
+        """
+        Delete the file 'mylocal.ip' (if exists) which is used to run
+        the gRPC in a custom ip.
+        """
+        mapdl_path = self.directory
+        if mapdl_path:
+            mylocalip_file = os.path.join(mapdl_path, 'mylocal.ip')
+            if os.path.isfile(mylocalip_file):
+                try:
+                    os.remove(mylocalip_file)
+                except OSError:
+                    pass
```

Comment:
```suggestion
```
While I'm a big fan of purity, adding this is one more thing to break, especially when the MAPDL already generates lots of temporary files that the user will probably clean up.  You can ignore this, but I'm up for removing this.

If you do keep it around, add in a check to return early if local.  For example:
```py
# exit early if not local
if not self._local:
    return
```

Reason being, we can't access the file if the server is remote and we might as well not even try.

Reply:
I think your reasoning is good. I will delete it.

Reply:
> Reason being, we can't access the file if the server is remote and we might as well not even try.

Thinking... if we cannot access the file, because the server is remote.... I'm not sure we can create the file at all. `mylocal.ip` is a file we create before running the ANSYS process. I haven't taken into account remote server execution when I coded this. 
Not sure if I should do something to make it work in that case.